src/managed/Microsoft.DotNet.PlatformAbstractions/RuntimeEnvironment.cs

```diff
@@ -60,6 +60,8 @@ private static string GetRIDVersion()
                     return $".{OperatingSystemVersion}";
                 case Platform.Darwin:
                     return $".{OperatingSystemVersion}";
+                case Platform.FreeBSD:
+                    return $".{OperatingSystemVersion.Split('.')[0]}";
```

Comment:
This is a little inconsistent with our other platforms.  For example, if you are on a RedHat 7.3 machine, `RuntimeEnvironment.OperatingSystemVersion` will return `7` and the RID is `rhel.7-x64`.  But on FreeBSD, `RuntimeEnvironment.OperatingSystemVersion` will return `11.1` and the RID is freebsd.11-x64.

Basically - the RID and the `OperatingSystemVersion` match on all our other platforms.  But here they aren't matching.

Reply:
I agree with @eerhardt, the OperatingSystemVersion should return 11 only here.

Reply:
I can fix that. FreeBSD has also text in version file like RELEASE or STABLE 
In my case the version is 11.0-RELEASE-p1.
Maybe for core-setup/cli we can keep the version consistent as you describe and have Platform functions to return true version as recognized by OS.

Would that be reasonable?

Reply:
Actually, everywhere (corefx, coreclr, core-setup), we use the normalized version only. So for RHEL, we have versions 6 and 7, although they are reported as 7.x and 6.x by the OS. Similar for Alpine, their version is 3.6.2, but we report 3.6. Basically, if the normalized version changes, it means a major OS change happened and not just minor update that fixes bugs, but keeps stuff compatible.